specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2023-09-01/examples/PutLogicalNetwork.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "subscriptionId": "fd3c3665-1729-4b7b-9a38-238e83b0f98b",
+    "resourceGroupName": "test-rg",
+    "logicalNetworkName": "test-lnet",
+    "api-version": "2023-09-01",
+    "logicalNetworks": {
+      "location": "West US2",
+      "extendedLocation": {
+        "type": "CustomLocation",
+        "name": "/subscriptions/fd3c3665-1729-4b7b-9a38-238e83b0f98b/resourceGroups/dogfoodarc/providers/Microsoft.ExtendedLocation/customLocations/dogfood-location"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/fd3c3665-1729-4b7b-9a38-238e83b0f98b/resourceGroups/dogfoodarc/providers/Microsoft.AzureStackHCI/logicalNetworks/test-lnet",
+        "name": "test-lnet",
+        "type": "Microsoft.AzureStackHCI/logicalNetworks",
+        "location": "West US2",
+        "extendedLocation": {
+          "name": "/subscriptions/fd3c3665-1729-4b7b-9a38-238e83b0f98b/resourceGroups/dogfoodarc/providers/Microsoft.ExtendedLocation/customLocations/dogfood-location",
+          "type": "CustomLocation"
+        },
+        "properties": {
+          "provisioningState": "Accepted"
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "id": "/subscriptions/fd3c3665-1729-4b7b-9a38-238e83b0f98b/resourceGroups/dogfoodarc/providers/Microsoft.AzureStackHCI/logicalNetworks/test-lnet",
+        "name": "test-lnet",
+        "type": "Microsoft.AzureStackHCI/logicalNetworks",
+        "location": "West US2",
+        "extendedLocation": {
+          "name": "/subscriptions/fd3c3665-1729-4b7b-9a38-238e83b0f98b/resourceGroups/dogfoodarc/providers/Microsoft.ExtendedLocation/customLocations/dogfood-location",
+          "type": "CustomLocation"
```

Comment:
Yes since it was defined the same way for [scvmm](https://github.com/Azure/azure-rest-api-specs-pr/blob/2c81cf086a071cde7497c04c946c2985bf249dd5/specification/scvmm/resource-manager/Microsoft.ScVmm/preview/2023-04-01-preview/examples/CreateVirtualMachine.json) and [vmware](https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/connectedvmware/resource-manager/Microsoft.ConnectedVMwarevSphere/preview/2023-03-01-preview/examples/CreateVirtualMachine.json) as well, which all fall under the same arc resource model. I'll mark it as won'tfix similar to the reason from previous comment. 

#won'tfix